test/test_katello/test_packages.py

```diff
@@ -9,10 +9,10 @@
 
 
 class TestUploadPackageProfile(TestCase):
+    @patch('katello.packages.plugin_enabled', return_value=True)
     @patch('katello.packages.get_manager')
     @patch('katello.packages.lookup_consumer_id')
-    @patch('katello.packages.PACKAGE_PROFILE_PLUGIN_CONF', 'test/test_katello/data/plugin_conf/enabled.conf')
-    def test_upload_registered(self, mock_lookup, mock_manager):
```

Comment:
I've updated tests which were calling into `plugin_enabled` to simply mock the value since we have adequate testing of the utils code already. I had to come here to add yet more mocks as a result of my change and this avoids that